app/models/contact.rb

```diff
@@ -569,4 +569,27 @@ def registrant?
   def deletable?
     !linked?
   end
+
+  def ident_human_description
+    description = "[#{ident_country_code} #{ident_type}]"
+    description.prepend("#{ident} ") if ident.present?
+
+    description
+  end
+
+  def as_csv_row
+    [
+      name,
+      code,
+      ident_human_description,
+      email,
+      created_at.to_formatted_s(:db),
+      registrar,
+      phone,
+    ]
+  end
+
+  def self.csv_header
+    ['Name', 'ID', 'Ident', 'E-mail', 'Created at', 'Registrar', 'Phone'].freeze
+  end
```

Comment:
Here is the same, you has identical principles in as_csv_row method and in csv_header, but csv_header looks like you tried to put there constant value. Why?